ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -93,22 +93,23 @@ class SqlLedgerSpec
       for {
         ledger <- createSqlLedger()
       } yield {
-        def allocateParty(displayName: String): Unit = {
+        def listPackages(displayName: String): Unit = {
           Await.result(
-            ledger.allocateParty(PartyIdGenerator.generateRandomId(), Some(displayName)),
-            patienceConfig.timeout)
+            ledger.listLfPackages(),
+            patienceConfig.timeout
+          )
           ()
         }
 
-        allocateParty("Alice")
+        listPackages("Alice")
         withClue("after allocating Alice,") {
```

Comment:
If you remove the display names, I'd recommend also adjusting the clues.

Reply:
Addressed in 165899128.